### PR TITLE
Make Modal component accept ref

### PR DESCRIPTION
**Pull request by christianaurichzm**

### CODE REVIEW

**Review by gustavomoser — DISMISSED** [review]
Nice!! :clap:

File: src/components/Modal/ModalContainer.tsx [hunk 1]
```diff
@@ -8,18 +8,19 @@ import { Icon } from '../Icon'
...
 
 export function ModalContainer(props: ModalContainerProps) {
-  const { style, onClose, hasCloseIcon, children, ...rest } = props
+  const { style, onClose, hasCloseIcon, children, containerRef, ...rest } = props
   const { classes, css } = useStyles(styles)
   const locale = useLocale()
 
   return (
-    <div role='dialog' aria-modal='true' className={css(classes.wrapper, style)} {...rest}>
+    <div role='dialog' aria-modal='true' ref={containerRef} className={css(classes.wrapper, style)} {...rest}>
```
Comment (Andrevmatias) [inline src/components/Modal/ModalContainer.tsx:23]: Esse não seria o caso de usar `fowardRef`: https://pt-br.reactjs.org/docs/forwarding-refs.html

**Review by Andrevmatias — CHANGES_REQUESTED** [review]
Improvement suggestion :blush:

File: src/components/Modal/ModalContainer.tsx (hunk 1 above)
Comment (christianaurichzm) [reply src/components/Modal/ModalContainer.tsx:23]: I had started using it, but gave up to keep the current design pattern of passing ref as prop. But from what I saw, some components are being refactored and starting to use forwardRef, so I think I will apply the suggestion.

Comment (christianaurichzm) [reply src/components/Modal/ModalContainer.tsx:23]: The only question I have is about something I read on this forwardRef API page.
`"When you start using forwardRef in a component library, you should treat it as a breaking change and release a new major version of your library."`
Are we sure about that yet?

Comment (christianaurichzm) [reply src/components/Modal/ModalContainer.tsx:23]: The only question I have is about something I read on that forwardRef API page.
`"When you start using forwardRef in a component library, you should treat it as a breaking change and release a new major version of your library."`
Are we sure about that yet?

Comment (Andrevmatias) [reply src/components/Modal/ModalContainer.tsx:23]: I don't see any changes in the bold components API: it's just an additional prop. It won't be necessary to change any code that already uses the Modal component after this change, right? Besides that, bold is still at a beta version, I don't think this change justifies a new major version.

Comment (christianaurichzm) [reply src/components/Modal/ModalContainer.tsx:23]: Nice! Thanks for the answer!

**Review by Andrevmatias — APPROVED** [review]
:rocket:

**Pull request merged**